common.mk

```diff
@@ -113,7 +116,7 @@ optee-os-common:
 
 OPTEE_OS_CLEAN_COMMON_FLAGS ?= $(OPTEE_OS_COMMON_FLAGS)
 
-optee-os-clean-common:
+optee-os-clean-common: xtest-clean
```

Comment:
You're right  :(

I propose to postpone xtest-clean to be fixed for another release, but to leave this `optee-os-clean-common: xtest-clean` for the time-being as it makes work 1 `make all-clean` at least